tests/unit/test_stac_utilities.py

```diff
@@ -271,3 +272,32 @@ def test_get_item_date_range(self):
         with self.subTest('Start and end datetimes'):
             self.assertTupleEqual(get_item_date_range(start_end_item),
                                   (self.datetime_one, self.datetime_two))
+
+    def test_is_netcdf_asset(self):
```

Comment:
Where is this test called?

Reply:
Good question! This is all part of the way `unittest` runs. The command to run the tests is:

```
coverage run --source harmony_netcdf_to_zarr -m unittest discover -s tests -t ..
```

There's a fair bit going on here, but the important part is `unittest discover -s tests`. That's essentially saying: "look for all tests in the `tests` directory". The `unittest` command will then scan files in that directory for instances of the `unittest.TestCase` class. For all those found, it will run the class methods that start with `test_`. [Here's a bit of documentation on `unittest` test discovery](https://docs.python.org/3/library/unittest.html#unittest-test-discovery).

In this specific instance, we can also compare the test artefacts from this PR and the last time something merged to `main`:

* The GitHub workflow that ran for this PR [is here](https://github.com/nasa/harmony-netcdf-to-zarr/actions/runs/1953963447). You can download either the artefact for  "test result" (Python 3.8 or Python 3.9). That will give you a `.zip` file. Once unzipped, there's a text file with all the logging from running the tests. The important thing there is that it should say "Ran 38 tests in XXXX".
* The GitHub workflow that ran the tests when last merging to `main` [is here](https://github.com/nasa/harmony-netcdf-to-zarr/actions/runs/1910915835). Again, you can download one of the "test result" artefacts, and open the resulting text file. That should say "Ran 37 tests in XXXX".

This PR adds a single extra test, so those two artefacts are consistent with that difference, showing that we ran one new test this time around.

Reply:
Oh okay - its all automagically done. Thanks for the thorough explanation.